bundles/org.openhab.ui/doc/components/oh-plan-marker.md

```diff
@@ -50,6 +50,13 @@ A marker on a floor plan
 <div class="props">
 <PropGroup name="icon" label="Icon">
   You can customize the styles further with CSS attributes in the <code>iconStyle</code> parameter (in YAML only)
+Example: (note that you must not have semicolons at the end of the css style):
+
+    iconStyle:
+      padding: 10px
+      border-radius: 4px
+      border: 5px solid #f00  
+
```

Comment:
You're not supposed to add stuff between the "GENERATED" tags in the docs directly, they would be overridden when those sections are regenerated using [this procedure](https://github.com/openhab/openhab-webui/blob/main/bundles/org.openhab.ui/CONTRIBUTING.md#documentation--resources).

Besides, chances are these wouldn't look good in the docs without code fences.

You can add examples at the end the file directly though - see oh-button.md.

Reply:
> You're not supposed to add stuff between the "GENERATED" tags in the docs directly, they would be overridden when those sections are regenerated using [this procedure](https://github.com/openhab/openhab-webui/blob/main/bundles/org.openhab.ui/CONTRIBUTING.md#documentation--resources).

That's a good hint. I tried to find a way to pregenerate the files similar like in the doc project before submitting the PR but didn't anything. Is there something I could run?


Regarding the code fences: I searched how to provide codes in md-files and https://www.markdownguide.org/basic-syntax/#code-blocks says it would be normale 4 spaces, which is why I did that way but no worries, sure I can use these.

> You can add examples at the end the file directly though - see oh-button.md.

Sure, will do

Reply:
I changed to the documentation. I hope it is fine now.